### PR TITLE
removed unused dependencies

### PR DESCRIPTION
**Pull request by julien51**
This removes a bunch of unused dependencies inside unlock-app, most of them needed for the paywall.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by benwerd** [comment]
(Sorry @cellog - misread and didn't mean to step on toes here)

File: unlock-app/src/createUnlockStore.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { createStore, applyMiddleware, combineReducers, compose } from 'redux'
+
```
Comment (cellog) [inline unlock-app/src/createUnlockStore.js]: ?

File: unlock-app/package.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
     "@storybook/addon-knobs": "^4.1.13",
     "@storybook/addon-storyshots": "^4.1.13",
     "@storybook/addon-viewport": "^4.1.13",
-    "@storybook/addons": "^4.1.13",
```
Comment (cellog) [inline unlock-app/package.json]: I'm surprised this is not used!

File: unlock-app/package.json [hunk 3]
```diff
@@ -45,21 +42,14 @@
...
     "react-test-renderer": "16.8.3",
     "react-testing-library": "^6.0.0",
     "redux": "^4.0.1",
-    "rollup": "^1.6.0",
-    "rollup-plugin-babel": "^4.3.2",
-    "rollup-plugin-terser": "^4.0.4",
     "rss": "^1.2.2",
     "run-script-os": "^1.0.5",
     "storybook-chromatic": "^1.3.0",
-    "storybook-react-router": "^1.0.2",
```
Comment (cellog) [inline unlock-app/package.json]: could you explain why this is not needed any longer? The last time I checked, storybook crashed if it wasn't present

File: unlock-app/package.json [hunk 4]
```diff
@@ -71,9 +61,7 @@
     "@svgr/cli": "^4.1.0",
     "depcheck": "^0.7.1",
     "nodemon": "^1.18.9",
-    "npm-check": "^5.9.0",
-    "react-router-test-context": "^0.1.0",
-    "web3-eth-abi": "1.0.0-beta.37"
+    "npm-check": "^5.9.0"
```
Comment (cellog) [inline unlock-app/package.json]: can we move that to dev deps?

Comment (julien51) [reply unlock-app/package.json]: it is in there ;) click the expand thing above!

File: unlock-app/package.json (hunk 2 above)
Comment (julien51) [reply unlock-app/package.json]: me too! which makes me think a lot more stuff is unused!

File: unlock-app/package.json (hunk 3 above)
Comment (julien51) [reply unlock-app/package.json]: I think we moved away a little while ago from that router... to use `connected-next-router`.
I tested Storybook and it works as expected..

**Pull request merged**